backends/platform/sdl/amigaos/amigaos.cpp

```diff
@@ -149,6 +152,14 @@ void OSystem_AmigaOS::initBackend() {
 	if (!ConfMan.hasKey("extrapath")) {
 		ConfMan.setPath("extrapath", "extras/");
 	}
+	if (!ConfMan.hasKey("iconspath")) {
+		ConfMan.setPath("iconspath", "icons/");
+	}
+	if (!ConfMan.hasKey("pluginspath")) {
+		ConfMan.setPath("pluginspath", "plugins/");
+	}
+	if (!ConfMan.hasKey("savepath")) {
+		ConfMan.setPath("savepath", "saves/");
```

Comment:
You miss a closing brace here.
I am also not sure these checks are needed.
I never understood the difference between `hasKey`/`setPath` and `registerDefault`.

Reply:
thank you, fixed